Core/ControlMapper.cpp

```diff
@@ -253,20 +263,27 @@ bool ControlMapper::UpdatePSPState(const InputMapping &changedMapping) {
 		float threshold = 1.0f;
 		bool touchedByMapping = false;
 		float value = 0.0f;
-		for (auto &mapping : inputMappings) {
-			if (mapping == changedMapping) {
+		for (auto &multiMapping : inputMappings) {
+			if (multiMapping.mappings.contains(changedMapping)) {
```

Comment:
Should this also check `if (multiMapping.empty())`?

-[Unknown]

Reply:
They're not supposed to ever be empty once mapped (we delete the whole thing from the keymap instead).

But I should add more asserts around to ensure that..

Reply:
Oh, was just looking at symmetry with the previous if block which did that check.

-[Unknown]

Reply:
Right, hehe.